lib/web/command.go

```diff
@@ -275,9 +286,96 @@ func (h *Handler) executeCommand(
 
 	runCommands(hosts, runCmd, h.log)
 
+	// Optionally try to compute the command summary.
+	if output := buffer.Export(); output != nil {
+		// Convert the map nodeId->output into a map nodeName->output
+		namedOutput := outputByName(hosts, output)
+
+		summary, err := h.summarizeOutput(namedOutput, clt, identity, req, ctx)
+		if err != nil {
+			h.log.Warn(err)
+			return nil, nil
```

Comment:
IMO, the error should be returned and logged by the caller, not the callee. Otherwise, the caller will not be aware that something has gone wrong.